src/uwtools/drivers/forecast.py

```diff
@@ -121,15 +129,92 @@ def requirements(self) -> None:
         ???
         """
 
-    def resources(self) -> None:
+    def resources(self, platform: dict) -> Mapping:
         """
-        ???
+        Set up the formatting for the scheduler batch script.
         """
-
-    def run(self):
+        # Add required fields to platform.
+        # Currently supporting only slurm and none.
+        slurmqueues = ["hera", "jet", "orion", "stampede"]
+        if platform["queue"] is None and platform["machine"] in slurmqueues:
+            platform["queue"] = "slurm"
+        else:
+            platform["queue"] = "none"
+
+        return {
+            "account": platform["account"],
+            "nodes": 1,
+            "queue": platform["queue"],
+            "scheduler": platform["scheduler"],
+            "tasks_per_node": 1,
+            "walltime": "00:01:00",
+        }
+
+    def run(self, log: Optional[Logger] = None) -> None:
         """
         Runs FV3.
```

Comment:
```suggestion
        Runs FV3 either as a subprocess or by submitting a batch script.
```

Reply:
OMG I didn't know you could make suggestions like that. Brilliant.

Reply:
oh wow, nice! However, it hid this whole block of suggestions from me and I only now saw it so thats a quirk to be aware of 0.o